tests/software_tests/can/packet/test_can_packet_record.py

```
@@ -0,0 +1,253 @@
+import pytest
+from mock import Mock, patch
+
+from uds.can import CanFlowStatus, CanPacketType
+from uds.can.packet.can_packet_record import (
+    AddressingType,
+    CanAddressingFormat,
+    CanPacketRecord,
+    PythonCanMessage,
+    ReassignmentError,
+    TransmissionDirection,
+    datetime,
```

Ruff (0.12.2)

Redefinition of unused `datetime` from line 1

(F811)